assets/js/blocks/product-collection/inspector-controls/index.tsx

```diff
@@ -50,6 +58,14 @@ const ProductCollectionInspectorControls = (
 				<OnSaleControl { ...props } />
 				<StockStatusControl { ...props } />
 				<KeywordControl { ...props } />
+				<AttributesControl
+					woocommerceAttributes={
+						props.attributes.query &&
+						( props.attributes.query
+							.woocommerceAttributes as AttributeMetadata[] )
+					}
+					setQueryAttribute={ setQueryAttributeBind }
```

Comment:
I'm not sure to understand `setQueryAttributeBind`, is it equivalent to this?

```suggestion
					setQueryAttribute={ ( newQuery ) => {
						setQueryAttribute( props, newQuery );
					} }
```

Reply:
Yes, You are correct. Do you have any concerns regarding the approach? 🙂

Reply:
Not really, just a personal preference as I find `bind` a bit difficult to understand. :smile: